node/src/main/resources/reference.conf

```diff
@@ -99,6 +99,9 @@ rnode {
     }
 
     reporting = false
+
+    # the max block number limit aquire from api
+    api-max-blocks-limit = 50
```

Comment:
@nzpr what do you think default should be?

For users to get data from validators very small number is enough.